src/porepy/grids/refinement.py

```diff
@@ -243,28 +245,18 @@ def remesh_1d(g_old, num_nodes, tol=1e-6):
     g.compute_geometry()
 
     # map the tags from the old grid to the new one
-
-    # retrieve the old faces and the corresponding coordinates
-    old_frac_faces = np.where(g_old.tags["fracture_faces"].ravel())[0]
-
-    # compute the mapping from the old boundary to the new boundary
-    # we need to go through the coordinates
-
-    new_frac_face = []
-
-    for fi in old_frac_faces:
-        nfi = np.where(cg.dist_point_pointset(g_old.face_centers[:, fi], nodes) < tol)[
-            0
-        ]
-        if len(nfi) > 0:
-            new_frac_face.append(nfi[0])
-
-    # This can probably be made more elegant
-    g.tags["fracture_faces"][new_frac_face] = True
-
-    # Fracture tips should be on the boundary only.
-    if np.any(g_old.tags["tip_faces"]):
-        g.tags["tip_faces"] = g.tags["domain_boundary_faces"]
+    # normally the tags are given at faces/point that are fixed the 1d mesh
+    # we use this assumption to proceed.
```

Comment:
Should this assumption be documented?

Reply:
what do you propose?

Reply:
Documentation in the function header? If it really is a basic assumption, it should not be tacit

Reply:
OK, I'll do it.